build.gradle.kts

```diff
@@ -321,6 +321,9 @@ tasks.register<org.openapitools.generator.gradle.plugin.tasks.GenerateTask>("ope
   outputDir.set("$projectDir/build/generated")
   apiPackage.set("it.gov.pagopa.nodo.gpd.controller.generated")
   modelPackage.set("it.gov.pagopa.nodo.gpd.dto.generated")
+  typeMappings.set(mapOf(
+    "DateTime" to "String" //GPD needs LocalDateTime instead of OffsetDateTime autogenerated
```

Comment:
```suggestion
    "DateTime" to "String" // GPD needs LocalDateTime instead of OffsetDateTime autogenerated, mapping to String because, for the same reason, we configured the ObjectMapper in order to always transform LocalDateTime into OffsetDateTime
```